README.rst

```diff
@@ -247,6 +247,53 @@ in terms of disk space.
 
 Whether ping checks are created automatically for devices.
 
+``OPENWISP_MONITORING_AUTO_DEVICE_CONFIGURATION_MODIFIED``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    | ``bool``    |
++--------------+-------------+
+| **default**: | ``True``    |
++--------------+-------------+
+
+``config_modified`` checks are run peridically to monitor whether device
+configuration is applied properly.
+This setting is used to govern whether ``config_modified`` checks are
+created automatically for newly registered devices.
+
+``OPENWISP_MONITORING_DEVICE_CONFIGURATION_MODIFIED_MAXIMUM_TIME``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-----------+
+| **type**:    |  ``int``  |
++--------------+-----------+
+| **default**: |   ``5``   |
++--------------+-----------+
+
+This setting allows you to configure the time that the check is allowed to
+fail after which the device health status changes to ``problem``.
+
+This default **duration** is 5 minutes. The input represents corresponding
+duration in minutes.
+
+**Note**: If the setting ``OPENWISP_MONITORING_AUTO_DEVICE_CONFIGURATION_MODIFIED``
+is disabled then this setting need not be declared.
+
+``OPENWISP_MONITORING_DEVICE_CONFIGURATION_MODIFIED_RETENTION_POLICY``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    |  ``time``   |
++--------------+-------------+
+| **default**: | ``48h0m0s`` |
++--------------+-------------+
+
+This setting allows to modify the duration for which the metric data generated
+by ``config_modified`` check is to be retained.
+
+**Note**: If the setting ``OPENWISP_MONITORING_AUTO_DEVICE_CONFIGURATION_MODIFIED``
+is disabled then this setting need not be declared.
+
```

Comment:
I think we should start creating a section which describes the available checks, which can go above, for now we'll have two checks: ping and this new one, we need a description for each one, and each section can then link the available settings for each check.

I would like to see a prose description of this new check.